src/go/util/request_test.go

```diff
@@ -39,30 +42,45 @@ func initServerForTestCallWithAccessToken(t *testing.T, desc, wantMethod, wantTo
 			t.Errorf("test(%v) fail, want Content-Type: application/x-protobuf, get Content-Type: %s", desc, got)
 		}
 
+		if respStatusCode != 0 && rejectCnt < rejectTimes {
```

Comment:
Suggest you re-use our generic retryHandler instead of writing your own code: https://github.com/GoogleCloudPlatform/esp-v2/blob/22f9d24a343b23176e350cf08e65f06dc4f5c4fd/tests/env/components/retry_handler.go#L41

Reduces code duplication.

Reply:
Prefer to keep the current version as it is a very simple test server, which is shared by all the testscase here and using this generic server also introduce other codes to config it.